frontend/resourceadm/components/AccessListMembers/AccessListMembers.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Alert, Textfield, Radio } from '@digdir/design-system-react';
+import classes from './AccessListMembers.module.css';
+import { AccessList, AccessListMember } from 'app-shared/types/ResourceAdm';
+import { FieldWrapper } from '../FieldWrapper';
+import { useRemoveAccessListMemberMutation } from '../../hooks/mutations/useRemoveAccessListMemberMutation';
+import { useAddAccessListMemberMutation } from '../../hooks/mutations/useAddAccessListMemberMutation';
+import { useDebounce } from 'react-use';
+import { usePartiesRegistryQuery } from '../../hooks/queries/usePartiesRegistryQuery';
+import { useSubPartiesRegistryQuery } from '../../hooks/queries/useSubPartiesRegistryQuery';
+import { getPartiesQueryUrl } from '../../utils/urlUtils';
+import { StudioSpinner } from '@studio/components';
+import { PlusIcon, PlusCircleIcon, MinusCircleIcon } from '@studio/icons';
+import { AccessListMembersPaging } from './AccessListMembersPaging';
+import { StudioButton } from '@studio/components';
```

Comment:
This should be imported together with the other import from @studio/components three lines up